website/docs/usage/training.md

```diff
@@ -33,6 +34,61 @@ ready-to-use spaCy pipelines.
 
 </Infobox>
 
+## Preparing Training Data {#training-data}
```

Comment:
I think we should add this section a little further down, at least below the quickstart? The quickstart is very relevant and probably what people should look at first, so I don't want this to move too far down on the page.

Reply:
I agree, maybe inbetween "config" an "custom training"?

Reply:
Sounds good! Placement was the main thing I struggled with here, because you need data before training but the Quickstart is the most important thing. I will try moving it there.